data/text/abilities.ts

```diff
@@ -0,0 +1,1480 @@
+export const AbilitiesText = {
+	noability: {
+		name: "No Ability",
+		shortDesc: "Does nothing.",
+	},
+	adaptability: {
+		name: "Adaptability",
+		desc: "This Pokemon's moves that match one of its types have a same-type attack bonus (STAB) of 2 instead of 1.5.",
+		shortDesc: "This Pokemon's same-type attack bonus (STAB) is 2 instead of 1.5.",
+	},
+	aerilate: {
+		name: "Aerilate",
+		desc: "This Pokemon's Normal-type moves become Flying-type moves and have their power multiplied by 1.2. This effect comes after other effects that change a move's type, but before Ion Deluge and Electrify's effects.",
+		shortDesc: "This Pokemon's Normal-type moves become Flying type and have 1.2x power.",
+		descGen6: "This Pokemon's Normal-type moves become Flying-type moves and have their power multiplied by 1.3. This effect comes after other effects that change a move's type, but before Ion Deluge and Electrify's effects.",
+		shortDescGen6: "This Pokemon's Normal-type moves become Flying type and have 1.3x power.",
```

Comment:
How does the `fieldN` approach scale to other mods (like Stadium or Lets Go even)?

Reply:
It doesn't.

Reply:
You could do e.g. `descGen1Stadium`, `descGen7LetsGo`.

Reply:
Underscore followed by formatid is certainly an option, though. It would break existing naming conventions but it would be easy to parse and be compatible with other formats.

Reply:
```ts
type Text = {
  name?: string;
  desc?: string;
  shortDesc?: string;
};
type Texts = Text & {[mod]: Text}
```

I dont think nesting is prohibitive

Reply:
Do you want to write a conversion script and/or find someone to manually do the conversion?

Reply:
Did you not write this with a script already? I would have imagined it could just be edited? If not, its not hard, I can write one. Appending `6` or `Gen6` or `_Gen6` is just much harder to make typesafe:

```js
const after = {};
for (let key in before) {
  const b = before[key];
  const a = after[key] = {};
  for (let k in b) {
    if (['name', 'desc', 'shortDesc'].includes(k)) {
      a[k] = b[k];    
    } else {
     const g = // TODO get last char
     k = // remove last characer - k.slice(0, -1) or something?
     const mod = `gen${g}`;
     a[mod] = a[mod] || {};
     a[mod][k] = b[k];
    }
  }
} 
```

Reply:
That's a value converter, not a file converter; it would wreck the comments...

Reply:
(I wrote a script to fill the current-gen data, and then had Flare manually fill in the past-gen data.)